package.json

```diff
@@ -38,13 +38,10 @@
     "@types/react": "^16.3.13",
     "@types/react-dom": "^16.0.5",
     "angular": "~1.5.8",
-    "angular-base64-upload": "^0.1.23",
```

Comment:
o_O Where it was used?

Reply:
In DynamicForm. We didn't use it for a long time now...